web/src/message_view_header.ts

```diff
@@ -38,19 +40,29 @@ function get_message_view_header_context(filter: Filter | undefined): MessageVie
     if (recent_view_util.is_visible()) {
         return {
             title: $t({defaultMessage: "Recent conversations"}),
+            description: $t({defaultMessage: "Overview of ongoing conversations."}),
             zulip_icon: "recent",
+            link: $t({defaultMessage: "/help/recent-conversations"}),
```

Comment:
Don't tag links for translation; that doesn't make any sense -- the URL is the same for everyone regardless of language. These should just be fixed URLs.